### PR TITLE
proposed structure

### PR DESCRIPTION
**Pull request by dgavrischuk**

### CODE REVIEW

**Comment by gbmeuk** [comment]
@dgavrischuk, can you merge master into this branch.

**Comment by dgavrischuk** [comment]
Done

**Review by AlexBMes — CHANGES_REQUESTED** [review]
Update the main mcp directory to match this structure

File: tests/mcp_structure_test/mcp/gcp_ae_locals.tf [hunk 1]
```diff
@@ -0,0 +1,89 @@
+locals {
```
Comment (gbmeuk) [inline tests/mcp_structure_test/mcp/gcp_ae_locals.tf:1]: Why is there code in the tests directory like this? Won't it override the real code in the module we're trying to test?

**Review by gbmeuk — COMMENTED** [review]
I presume some of these test files were actually supposed to be updates to the main module?￼

**Review by gbmeuk — CHANGES_REQUESTED** [review]
see comments

**Review by AlexBMes — DISMISSED** [review]

**Comment by gbmeuk** [comment]
There was an HTTP path in one of the Kubernetes adapters that was defaulting on the platform to "/" and was shown as changing on every run as a result.
@AlexBMes @dgavrischuk

**Review by gbmeuk — APPROVED** [review]

**Pull request merged**